### PR TITLE
formatting

### PR DESCRIPTION
**Pull request by dorinhogea**
basically playing with isspace()

### CODE REVIEW

File: db/comdb2.c [hunk 1]
```diff
@@ -5715,8 +5715,7 @@ static struct dbenv *newdbenv(char *dbname, char *lrlname)
         return NULL;
     }
 
-    if (lrlname)
-        pre_read_lrl_file(dbenv, lrlname, dbname);
+    if (lrlname) pre_read_lrl_file(dbenv, lrlname, dbname);
```
Comment (adizaimi) [inline db/comdb2.c]: I wonder if we should leave the existing one line if stmts as is, ie. not put them in one line like here, but leave them like it was before.

**Comment by riverszhang89** [comment]
Thanks Dorin.

**Comment by dorinhogea** [comment]
thank you!

File: db/comdb2.c (hunk 1 above)
Comment (dorinhogea) [inline db/comdb2.c]: Yes, I would like that, it is easier to set a breakpoint on condition match if two lines. Change roborivs?

Comment (riverszhang89) [inline db/comdb2.c]: Roborivers uses the coding style in [this doc](https://bloomberg.github.io/comdb2/contrib.html#indentation-etc
).

Comment (adizaimi) [inline db/comdb2.c]: Well, you can try with AllowShortIfStatementsOnASingleLine: false, or try omitting it totaly.
I did not think we decided to force if stmts on same line -- if we did that's silly and we should not enforce.

**Pull request merged**